addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -52,31 +52,56 @@ type antreaFlowExporter struct {
 	IdleFlowTimeout   string `yaml:"idleFlowTimeout,omitempty"`
 }
 
+type antreaMultiCluster struct {
+	Enable    bool   `yaml:"enable,omitempty"`
+	Namespace string `yaml:"namespace,omitempty"`
+}
+
+type antreaIPsec struct {
```

Comment:
IPSec is removed from the Antrea commercial release image. This is because of US export regulation prevent us from distributing IPSec packages. Let's remove all of the IPSec options from AntreaConfig.